assets/css/index.css

```
@@ -151,46 +151,51 @@ iframe#twitter-widget-0 {
   height: 30px !important;
 }
 
-.github-fork-ribbon-wrapper {
-  width: 150px;
-  height: 150px;
-  position: fixed;
-  overflow: hidden;
+.github-corner {
+  fill: #151513;
+  color: #fff;
+  position: absolute;
```

I'd suggest that `position: fixed` would be preferable as @owenvoke suggests.
```suggestion
  position: fixed;
```